src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetProjectManagerService.cs

```diff
@@ -398,6 +399,19 @@ public async ValueTask ExecuteActionsAsync(IReadOnlyList<ProjectAction> actions,
             DependencyBehavior dependencyBehavior,
             IReadOnlyList<string> packageSourceNames,
             CancellationToken cancellationToken)
+        {
+            return await GetInstallActionsAsync(projectIds, packageIdentity, versionConstraints, includePrelease, dependencyBehavior, packageSourceNames, versionRange: null, cancellationToken);
+        }
+
+        public async ValueTask<IReadOnlyList<ProjectAction>> GetInstallActionsAsync(
+            IReadOnlyCollection<string> projectIds,
+            PackageIdentity packageIdentity,
+            VersionConstraints versionConstraints,
+            bool includePrelease,
+            DependencyBehavior dependencyBehavior,
+            IReadOnlyList<string> packageSourceNames,
+            VersionRange? versionRange,
```

Comment:
I tried deleting the `?` and it shows an error, `CS8625: Cannot convert null literal to non-nullable reference object`, not sure if I need to add a suppress to the file because in `PreviewProjectsInstallPackageAsync` I can use it without the `?`

Reply:
Ah, I see, we don't have `#nullable` enabled in `NuGetPackageManager.cs`.